app/src/main/java/com/keylesspalace/tusky/network/MastodonApi.kt

```diff
@@ -609,9 +609,7 @@ interface MastodonApi {
     ): NetworkResult<Poll>
 
     @GET("api/v1/announcements")
-    suspend fun listAnnouncements(
-        @Query("with_dismissed") withDismissed: Boolean = true
-    ): NetworkResult<List<Announcement>>
+    suspend fun announcements(): NetworkResult<List<Announcement>>
```

Comment:
This parameter does not exist https://github.com/mastodon/mastodon/issues/27310#issuecomment-1750914701